packages/@livestore/common/src/leader-thread/LeaderSyncProcessor.ts

```diff
@@ -275,6 +282,7 @@ export const makeLeaderSyncProcessor = ({
       }
 
       /** State transitions need to happen atomically, so we use a Ref to track the state */
+      yield* Effect.logDebug(`Init syncState: local=${initialSyncState.localHead.global}, upstream=${initialSyncState.upstreamHead.global}`)
```

Comment:
I added several `syncState` debug logs whenever the sequence numbers for local/remote change, it's not too much of logs IMO and it's on debug level but feel free to adjust. It was crucial during validating this